sys/net/application_layer/nanocoap/sock.c

```diff
@@ -56,15 +56,6 @@ static uint16_t _get_id(void)
     return atomic_fetch_add_u16(&id, 1);
 }
 
-int nanocoap_sock_connect(nanocoap_sock_t *sock, sock_udp_ep_t *local, sock_udp_ep_t *remote)
-{
-    if (!remote->port) {
-        remote->port = COAP_PORT;
```

Comment:
Technically this makes this an API change, but I couldn't find a user other than `nanocoap_sock_url_connect()` that relied on this.